app/controllers/organizations/staff/external_form_upload_controller.rb

```diff
@@ -4,18 +4,20 @@ class ExternalFormUploadController < Organizations::BaseController
       layout "dashboard"
 
       def index
-        authorize! :external_form_upload,
-          context: {organization: Current.organization}
+        authorize! :external_form_upload, context: {organization: Current.organization}
       end
 
       def create
-        authorize! :external_form_upload,
-          context: {organization: Current.organization}
-        file = params[:files]
+        authorize! :external_form_upload, context: {organization: Current.organization}
 
         # Only processes single file upload
-        import_service = Organizations::Importers::GoogleCsvImportService.new(file)
-        import_service.call
+        import = Organizations::Importers::GoogleCsvImportService.new(params[:files]).call
+
+        if import.success?
+          # do something
```

Comment:
Did you want to make an issue for handling these paths? 
I think we want to render a summary table of imports beneath the form input with turbo frame right?

Reply:
> Did you want to make an issue for handling these paths? I think we want to render a summary table of imports beneath the form input with turbo frame right? We could have a green 'Success' or a red 'Import Errors' or something along those lines.

Yes. I wanted to make sure this service was good to go before opening the issue for the front end/controller changes.